src/components/DropCliff.tsx

```diff
@@ -0,0 +1,62 @@
+import { css } from '../../styled-system/css'
+import Path from '../@types/Path'
+import { isTouch } from '../browser'
+import { HOME_PATH } from '../constants'
+import head from '../util/head'
+import unroot from '../util/unroot'
+import DropEnd from './DropEnd'
+
+/** Renders a drop target for dropping thoughts after a cliff. */
+const DropCliff = ({
+  cliff,
+  depth,
+  path,
+  isTableCol2,
+  prevWidth = 0,
+  isLastVisible,
+}: {
+  cliff: number
+  depth: number
+  path: Path
+  isTableCol2: boolean
+  prevWidth?: number
+  isLastVisible?: boolean
+}) => {
+  if (cliff < 0) {
```

Comment:
Minor point, but I tend to prefer the negation for short circuiting statements. It results in a smaller indent level and fewer curly braces to deal with:

```ts
if (cliff >= 0) return null

return ...
```